skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -566,29 +562,57 @@ const std::vector<espm::RecordHeader*>& espm::Browser::GetRecordsByType(
 }
 
 const std::vector<espm::RecordHeader*>& espm::Browser::GetRecordsAtPos(
-  uint32_t cellOrWorld, int16_t cellX, int16_t cellY)
+  uint32_t cellOrWorld, int16_t cellX, int16_t cellY) const
 {
   return pImpl->cellOrWorldChildren[RefrKey(cellOrWorld, cellX, cellY)];
```

Comment:
`operator[]` is non-const, that's why there was no `const modifier on the method here. The compiler doesn't handle it due to the pImpl idiom. Consider adding comment about this and apply one of the following fixes:

- Remove `const` back
- Reimplement method without `operator[]` (with `find`)

Reply:
I guess sparse map has const `operator[]`, otherwise it wouldn't compile. But it will be better with find I guess

Reply:
Couldn't find it though for some reason

Reply:
Now explicitly returning reference to a static variable